validate.js

```diff
@@ -0,0 +1,64 @@
+const form = document.getElementById('f1');
+const fname = document.getElementById('text1');
+const email = document.getElementById('email1');
+const message = document.getElementById('text_zone1');
+const errorName = document.getElementById('errorName');
+const errorEmail = document.getElementById('errorEmail');
+const errorMsg = document.getElementById('errorMsg');
+
+const overralError = document.getElementById('overralError');
+
+const emailRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+
+fname.addEventListener('input', () => {
+  if (fname.value.length < 30) {
+    errorName.textContent = '';
+  } else {
+    errorName.textContent = 'Name should be less than 30 characters.';
+  }
+});
+
+email.addEventListener('input', () => {
+  if (email.value === email.value.toLowerCase()) {
+    errorEmail.textContent = '';
+  } else {
+    errorEmail.textContent = 'Email Address should not have uppercase letters.';
+  }
```

Comment:
- [ ] Please make the error messages appear close to the submit button as per requirements, preferably in red so that users know it's an error message 👍 
![image](https://user-images.githubusercontent.com/46839548/190869802-796b5b40-3ed0-4e98-a4b2-cf68d861a10a.png)